src/components/UsageBar.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="header">
+    <slot name="header" />
+  </div>
+  <ProgressBar
+    v-for="(item, index) in items"
+    :badge-label="item.badgeLabel"
+    :key="index"
```

Comment:
The index doesn't work as a key since the items can be sorted differently on each render.